lib/gracenote/response/album.rb

```diff
@@ -1,24 +1,44 @@
+require "gracenote/response/base"
+require "gracenote/response/genre"
+require "gracenote/response/track"
+require "gracenote/response/cover_art"
+
 module Gracenote
   class Response
-    class Album
-      attr_reader :response
+    class Album < Base
+      include Helper
 
-      def initialize(response)
-        @response = response
-      end
+      self.attributes = %w[
```

Comment:
__[rubocop]__
* `%w`-literals should be delimited by `(` and `)`. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)